internal/server/backgroundjobs.go

```diff
@@ -51,6 +51,7 @@ func jobWrapper(ctx context.Context, db *data.DB, job BackgroundJobFunc, every t
 			if err != nil {
 				return fmt.Errorf("failed to start transaction :%w", err)
 			}
+			tx = tx.WithOrgID(db.DefaultOrg.ID)
```

Comment:
I suspect that most of our jobs will perform queries that ignore the organization, but for testing it helped to have the org set.  The previous code would set a default org, so this restores the behaviour from before #3643.

Reply:
Fair enough. I added it because the code I had refactored was doing this previously and I didn't want to change the behaviour yet.  

It's unfortunately somewhat risky I think, because so much of our code panics if we don't set an OrgID (which I'm not a fan of), and it's very difficult to test the integration of the "jobs" with this `jobWrapper`.

So we end up not having very good test coverage to catch when someone creates a job that expects an organization. I guess it's still better that this panics than to only use the default org. 

I'll remove this line and handle it in the test.